Integrations/integration-CrowdStrikeHost.yml

```diff
@@ -823,7 +937,7 @@ script:
     - name: first_behavior
       description: 'First Behavior of the detection. Example: 2017-01-31T22:36:11Z'
     outputs:
-    - contextPath: CrowdStrikeHost.Detections.detectionID
+    - contextPath: CrowdStrikeHost.Detections.detection_id
```

Comment:
@DeanArbel Was this a mistake in the outputs?

Reply:
Yeah